src/sql/development.clj

```diff
@@ -89,11 +90,11 @@
     (doseq [row [(new-address (new-uuid 300) "222 Other")
                  (new-account (new-uuid 100) "Tony" "tony@example.com" "letmein" salt iterations
                    :account/primary_address (new-uuid 300)
-                   :account/role ":account.role/superuser"
+                   :account/role (as-other ":account.role/superuser")
```

Comment:
I was not aware of this. Does this make this work in PostgreSQL and MySQL and others?  I'm not working in SQL much these days.